server/home/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
```

Comment:
Double newline. No idea what's the convention, should it be changed to one? In other files too.

Reply:
I think that PyCharm suggests that conventian but not sure

Reply:
PEP8: two newlines before defs, code was checked by linter in pycharm.